cellprofiler/modules/correctilluminationcalculate.py

```diff
@@ -206,209 +226,222 @@ def create_settings(self):
-            prior to dilation or smoothing. It is an image produced during the calculations, not typically
-            needed for downstream modules. It can be helpful to retain it in case you wish to try several
-            different smoothing methods without taking the time to recalculate the averaged image each time.
-            <p>Select <i>%(YES)s</i> to retain this averaged image. Use the <b>SaveImages</b> module to save
-            it to your hard drive.</p>''' % globals())
+            "Retain the averaged image?", False, doc="""\
+The averaged image is the illumination function prior to dilation or
+smoothing. It is an image produced during the calculations, not
+typically needed for downstream modules. It can be helpful to retain it
+in case you wish to try several different smoothing methods without
+taking the time to recalculate the averaged image each time.
+            
+Select *%(YES)s* to retain this averaged image. Use the **SaveImages**
+module to save it to your hard drive.
+""" % globals())
 
         self.average_image_name = cps.ImageNameProvider(
-                "Name the averaged image", "IllumBlueAvg", doc='''
-            <i>(Used only if the averaged image is to be retained for later use in the pipeline)</i><br>
-            Enter a name that will allow the averaged image to be selected later in the pipeline.''')
+            "Name the averaged image", "IllumBlueAvg", doc="""\
+*(Used only if the averaged image is to be retained for later use in the pipeline)*
+Enter a name that will allow the averaged image to be selected later in the pipeline.""")
 
         self.save_dilated_image = cps.Binary(
-                "Retain the dilated image?", False, doc='''
-            The dilated image is the illumination function after dilation but prior to smoothing.
-            It is an image produced during the calculations, and is not typically needed for downstream modules.
-            <p>Select <i>%(YES)s</i> to retain this dilated image. Use the <b>SaveImages</b> module to save it
-            to your hard drive.</p>''' % globals())
+            "Retain the dilated image?", False, doc="""\
+The dilated image is the illumination function after dilation but prior
+to smoothing. It is an image produced during the calculations, and is
+not typically needed for downstream modules.
+            
+Select *%(YES)s* to retain this dilated image. Use the **SaveImages**
+module to save it to your hard drive.
+""" % globals())
 
         self.dilated_image_name = cps.ImageNameProvider(
-                "Name the dilated image", "IllumBlueDilated", doc='''
-            <i>(Used only if the dilated image is to be retained for later use in the pipeline)</i><br>
-            Enter a name that will allow the dilated image to be selected later in the pipeline.''')
+            "Name the dilated image", "IllumBlueDilated", doc="""\
+*(Used only if the dilated image is to be retained for later use in the pipeline)*
```

Comment:
Add a newline between `*(Used only if...)*` and the next line. This will insert a line break in the rendered documentation. Otherwise, the text appears on the same line.